src/api/utils/subscription/targets.py

```diff
@@ -59,7 +59,9 @@ def assign_targets(sub_level):
         available_templates = get_target_available_templates(
             target["email"], history, sub_level["template_uuids"]
         )
-        selected_template = random.choice(available_templates)
+
+        # Ignore this bandit error, not used for security/cryptography purposes.
```

Comment:
For improved context, please include the name/id of the bandit error in question here.

Reply:
I have provided a link in the comments for B311 - random. https://bandit.readthedocs.io/en/latest/blacklists/blacklist_calls.html#b311-random